perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/registrartabs/EditMailFooterTabItem.java

```diff
@@ -224,11 +242,10 @@ public void open()
 	}
 
 	public boolean isAuthorized() {
-
 		if (session.isVoAdmin(voId)) {
 			return true;
 		} else {
-			return false;
+			return group != null && session.isGroupAdmin(groupId);
```

Comment:
Hi, since existing object ids start with 1 and you define voId/groupId=0 by default, then you can safely use something like: ``session.isVoAdmin(voId) || session.isGroupAdmin(groupId)`` in the first IF.